docs/source/reference/changelog.md

```diff
@@ -47,22 +55,86 @@ command line for details.
 - [OpenShift] {attr}`.OpenShiftOAuthenticator.validate_cert` is deprecated and
   is being replaced by {attr}`.OAuthenticator.validate_server_cert`.
 
-### Highlights
+#### A new structure
+
+The authenticators are no longer overriding the `authenticate` method, but
+instead relying on the OAuthenticator base class `authenticate` method which
+calls a few lower level methods that can be overridden if needed. Like this, a
+lot of code has been absorbed into the OAuthenticator base class that was
+previously duplicated across authenticators.
+
+To learn more about this new structure the provider specific authenticator
+classes rely on, please for now inspect the source code for the
+`OAuthenticator.authenticate` method and `OAuthenticator.check_allowed` method.
+Plans on writing more thorough documentation about this new structure is tracked
+in issue [#634](https://github.com/jupyterhub/oauthenticator/issues/634).
+
+#### New features added
+
+- breaking: add allow_existing_users config [#631](https://github.com/jupyterhub/oauthenticator/pull/631) ([@consideRatio](https://github.com/consideRatio), [@minrk](https://github.com/minrk))
```

Comment:
Should this be marked `[all]`?

Reply:
```suggestion
- [All] breaking: add allow_existing_users config [#631](https://github.com/jupyterhub/oauthenticator/pull/631) ([@consideRatio](https://github.com/consideRatio), [@minrk](https://github.com/minrk))
```

Reply:
Updated the PR title as well retroactively